.github/workflows/auto-format.yml

```
@@ -17,28 +17,52 @@ jobs:
       (github.event_name == 'push' || github.event.pull_request.head.repo.full_name == github.repository) &&
       github.actor != 'github-actions[bot]'
     steps:
       - uses: actions/checkout@v4
         with:
           ref: ${{ github.head_ref || github.ref_name }}
-          # Use PAT to trigger downstream workflows; fallback to GITHUB_TOKEN
+          path: meta_git_lib
           token: ${{ secrets.PARENT_REPO_PAT || github.token }}
 
+      - name: Clone dependencies
+        run: |
```

actionlint (1.7.10)

[error] "github.head_ref" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)